src/Forms/TextareaField.php

```diff
@@ -117,7 +140,8 @@ public function getAttributes()
                 'rows' => $this->getRows(),
                 'cols' => $this->getColumns(),
                 'value' => null,
-                'type' => null
+                'type' => null,
+                'maxlength' => $this->getMaxLength(),
```

Comment:
We should only set this attribute if non empty.

See TextField.php for pre-existing logic:

```php
if ($maxLength) {
    $attributes['maxLength'] = $maxLength;
    $attributes['size'] = min($maxLength, 30);
}
```

Reply:
Fixed.